package.json

```diff
@@ -53,37 +53,39 @@
     ]
   },
   "dependencies": {
-    "ccxt": "1.54.33",
+    "ccxt": "^1.56.59",
```

Comment:
this kind of dep can have a ton of side effects, that's why fixing versions may be a better approach, what are your thoughts about it ?

Reply:
Makes sense, and since this is built within docker, we'd like it to be predictable. I might revert the version changes and put that up in a separate pr

Reply:
Without thinking about it, I left `madge` at `^5.0.1`, but as a devDependency, I figured it wouldn't affect the build.